build_container.sh

```diff
@@ -35,6 +34,7 @@ baseimage=`grep FROM Dockerfile | sed -e 's/FROM //'`
 ${ENGINE_CMD} pull $baseimage
 
 ${ENGINE_CMD} build \
+       --build-arg TARGETPLATFORM=$TARGETPLATFORM \
```

Comment:
I'm not sure this is necessary. When I use a regular `docker build`, this gets defined.

```Dockerfile
from ubuntu:20.04

ARG TARGETPLATFORM
RUN echo $TARGETPLATFORM
```

```
$ docker --version
Docker version 20.10.12, build e91ed57
```

Reply:
This extends to the matrix additions and the `-builder` image below as well.

Reply:
I am not sure if `podman` or other supported container platforms do the same.

Reply:
```
FROM ubuntu:20.04

RUN env
```

```
HOSTNAME=<foo>
HOME=/root
PATH=<...>
PWD=/
```

So looks like `docker` implementations are not consistent... I had to pass in `--build-arg TARGETPLATFORM=$(uname -m)` to have something defined.

Reply:
If you were to use `docker buildx build` does that define `TARGETPLATFORM`? Since the build script supports other container platforms we probably couldn't remove it anyways since I don't think those platforms have the `buildx` command.